front/src/modules/simulationResult/helpers/formatTimetableItemSummaries.ts

```diff
@@ -1,68 +1,67 @@
-import formatBaseTimetableItemWithDetails from 'modules/timetableItem/helpers/formatBaseTimetableItemWithDetails';
-import type { TimetableItemId, TimetableItem } from 'reducers/osrdconf/types';
-import { Duration } from 'utils/duration';
-import { jouleToKwh } from 'utils/physics';
-import { formatKmValue } from 'utils/strings';
+import {
+  formatPacedTrainWithDetails,
+  formatTrainScheduleWithDetails,
+} from 'modules/timetableItem/helpers/formatTimetableItemWithDetails';
+import type {
+  TimetableItemId,
+  TimetableItem,
+  TrainScheduleId,
+  PacedTrainId,
+} from 'reducers/osrdconf/types';
+import { isPacedTrainResponseWithPacedTrainId } from 'utils/trainId';
 import { mapBy } from 'utils/types';
 
-/**
- * Format the timetable items with their simulation summaries
- */
+/** Format the timetable items with their simulation summaries */
 const formatTimetableItemSummaries = (
-  rawSummaries: Map<TimetableItemId, SimulationSummaryResult>,
+  rawTrainScheduleSummaries: Map<TrainScheduleId, SimulationSummaryResult>,
+  rawPacedTrainSummaries: Map<PacedTrainId, PacedTrainSimulationSummaryResult>,
   rawTimetableItems: Map<TimetableItemId, TimetableItem>,
   rollingStocks: LightRollingStockWithLiveries[]
 ): Map<TimetableItemId, TimetableItemWithDetails> => {
-  const items = [...rawSummaries].map(([id, timetableItemSummary]): TimetableItemWithDetails => {
-    const timetableItem = rawTimetableItems.get(id);
-    if (!timetableItem) {
+  const items: TimetableItemWithDetails[] = [];
+
+  // train schedules
+  [...rawTrainScheduleSummaries].forEach(([id, summary]) => {
+    const trainSchedule = rawTimetableItems.get(id);
+    if (!trainSchedule) {
       throw new Error('Missing timetable item');
     }
+    if (isPacedTrainResponseWithPacedTrainId(trainSchedule)) {
+      throw new Error('Timetable item should be a train schedule, but it is a paced train');
+    }
+    const rollingStock = rollingStocks.find((rs) => rs.name === trainSchedule.rolling_stock_name);
+    const trainScheduleWithDetails = formatTrainScheduleWithDetails(
+      trainSchedule,
+      rollingStock,
+      summary
+    );
 
-    let notHonoredReason: TimetableItemWithDetails['notHonoredReason'];
-    if (timetableItemSummary.status === 'success') {
-      if (isTooFast(timetableItem, timetableItemSummary)) notHonoredReason = 'trainTooFast';
-      if (isScheduledPointsNotHonored(timetableItem, timetableItemSummary))
-        notHonoredReason = 'scheduleNotHonored';
+    items.push(trainScheduleWithDetails);
+  });
+
+  // paced trains
+  [...rawPacedTrainSummaries].forEach(([id, pacedTrainSummary]) => {
+    const pacedTrain = rawTimetableItems.get(id);
+    if (!pacedTrain) {
+      throw new Error('Missing timetable item');
+    }
+    if (!isPacedTrainResponseWithPacedTrainId(pacedTrain)) {
+      throw new Error('Timetable item should be a paced train, but it is a train schedule');
```

Comment:
```suggestion
      throw new Error('Timetable item should be a paced train');
```
we just know its id doesn’t start with 'paced_' from the test we do. it could be any kind of error. better not assume what it is and avoid misleading when debugging

Reply:
Done